data/alexandria start.txt

```diff
@@ -0,0 +1,1304 @@
+				`	Change the subject...`
+					goto questions
+				`	"Sounds good.  Let's go."`
+					goto go
+				`	"I don't have time to help you any more."`
+					decline
+			label seeker
+			`"An Alexandria Seeker is an adventurer who scours the galaxy for works of literature or art and brings them back to the library for archival.  It's a dangerous job, but it is worth the risk to ensure knowledge is preserved.`
+			`	"When you get a pilot license, the paperwork goes to many organizations in the Republic.  Alexandria Library is one of them.  If your personality fits and you've show determination and resourcefulness, they consider you for an apprenticeship.  Finish your first mission, and you'll be a Journeyman Seeker.  Prove your skill, and you'll be a Master Seeker with a sizable salary.  We always Seek in pairs, so you'll still have to ferry someone even as a Master Seeker."`
+				goto "seeker questions"
+			label "why me"
+			`"I haven't read your file, but the reason is usually a mix of determination and resourcefulness based on your past five years of work.  There are basic educational requirements too, and a few other minor things."`
+				goto "seeker questions"
+			label faults
+			`"You must know yourself if you are to be a successful Seeker.  You are never truly strong until you turn your very weaknesses into strengths."`
+				goto "seeker questions"
+			label go
+			`"Excellent!  That's the spirit."`
+				accept
+	on decline
+		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
+		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
+	on fail
+		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
+		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
+	on visit
+		dialog `It seems you left the insane librarian or some of his thousand pencils behind.  Perhaps they're in one of your escorts?`
+	on complete
+		conversation
+			`Instead of landing on the main Amazon spaceport, Daemos has you land in a clearing deep inside one of the densest forests on the planet.  You climb through some aggressive vegetation, up a mountain, to a hut.  Daemos looks at you and says, "this is the place."`
+			choice
+				`	Open the door...`
+			`When the door opens, Daemos rushes forward in a panic, before you can even survey the room.  It seems the man you came to see is dead, and quite recently too.`
+			`	Daemos searches the room and finds the culprit.  The author was working on the second edition of his book, "A Thousand Poisons of Amazon and How to Conceal Their Taste."  Somehow, one of the poison recipes was swapped with the lunch recipe.  A poetic end to someone whose work has lead to many deaths in the Syndicate.`
+			`	This seems to be the end of the quest, so you ferry Daemos back to the main Amazon spaceport.`
+			choice
+				`	Fly back to the main spaceport...`
+			`"This will just delay us," says Daemos, "I know a group of businessmen on Deadman's Cove who can forge this signature."`
```

Comment:
The executive asked for a signed first-edition copy of a book.  Daemos had the book, but it was not signed.

Reply:
Wouldn't the correct word be "his" instead then?
```suggestion
			`"This will just delay us," says Daemos, "I know a group of businessmen on Deadman's Cove who can forge his signature."`
```